test/px-data-table-test.js

```diff
@@ -886,6 +886,29 @@ function runTests() {
       lastNameFilter.dispatchEvent(new Event('keyup'));
     });
 
+    test('When selecting all, select only what is filtered', function(done){
+
+      var tb = Polymer.dom(table2Fixture.root).querySelector('aha-table');
+      table2Fixture.tableData = table1Fixture.tableData;
```

Comment:
Is this here for a reason? If I remove it, the _validate error goes away from this failing test case. If it's because of the test immediately prior to this one, you can add two lines to the end of it to "put back" the filter to its prior state:

```
lastNameFilter.value = '';
lastNameFilter.dispatchEvent(new Event('keyup'));
```